Apps/APN-UIKit/APN UIKit/View/DashboardViewController.swift

```diff
@@ -36,6 +36,11 @@ class DashboardViewController: BaseViewController {
         // For inline Views added with Storyboard, set the elementId to finish setup of the View and begin showing messages.
         inlineInAppViewCreatedInStoryboard.elementId = "dashboard-announcement"
 
+        // This is optional. If set, the delegate method `onInlineCustomButtonAction`
+        // will receive callbacks.
+        // If not set, the global method `messageActionTaken` will handle the callbacks.
+        inlineInAppViewCreatedInStoryboard.inlineMessageDelegate = self
```

Comment:
The [tech proposal's public API](https://www.notion.so/custio/Mobile-inline-in-app-messages-technical-proposal-7efb22e2af1c40449c72f9cf73770584?pvs=4#d722b4ab9b854d84b86d744cd840d307) specifies the delegate name to be `onActionDelegate`. Could you rename the property name to match this?

Reply:
I didn't notice it. Thanks for pointing out.